browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -572,6 +572,24 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_BLOCK_TRACKERS_ADS_SUMMARY" desc="Text summary for Block trackers and ads.">
         Prevents trackers &amp; ads from loading and showing in webpages
       </message>
+      <message name="IDS_HTTPS_UPGRADE_TEXT" desc="Text for shields option.">
```

Comment:
we should use descriptive `desc` as it's useful for translation.

Reply:
fixed